JSDemos/Demos/VectorMap/CustomAnnotations/description.md

```diff
@@ -0,0 +1,9 @@
+Annotations are images and text blocks that provide additional information about the visualized data.
+
+To configure individual annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
+
+Each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#type) attribute should be set to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"custom"*.
+
+To create an annotation of this type, declare the SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#template) option. Inside the template, you can access the annotation being customized.
```

Comment:
To create an annotation of that type, you set its type to "custom" :-) In this case we already talked about type, so now we need to talk about specifying content for these custom annotations.  

```suggestion
Custom annotations require that you specify your own display [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#template) in SVG format. As you can see in demo code, you can access annotation data within the template markup.
```